recipe/build.sh

```diff
@@ -1,14 +1,40 @@
 #!/bin/bash
 
+# Remove the full path from CXX etc. If we don't do this
+# then the full path at build time gets put into 
+# mkspecs/qmodule.pri and qmake attempts to use this.
+export AR=$(basename ${AR})
+export RANLIB=$(basename ${RANLIB})
+export STRIP=$(basename ${STRIP})
+export OBJDUMP=$(basename ${OBJDUMP})
+export CC=$(basename ${CC})
+export CXX=$(basename ${CXX})
+
+if [[ ${HOST} =~ .*darwin.* ]]; then
+  # Avoid Xcode
+  cp "${RECIPE_DIR}"/xcrun .
+  cp "${RECIPE_DIR}"/xcodebuild .
```

Comment:
@ocefpaf do we need all this code? Does it fail otherwise? I would have thought this was only needed for `qmake`, not `cmake`

Reply:
To be honest I don't know and I don't think we need them. I was just trying to copy everything over from the `qt` feedstock to see if I could get this one to build on macOS.